packages/docusaurus-utils/src/__tests__/urlUtils.test.ts

```diff
@@ -5,12 +5,12 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import {addTrailingSlash} from '@docusaurus/utils-common';
 import {
   normalizeUrl,
   getEditUrl,
   fileToPath,
   isValidPathname,
-  addTrailingSlash,
   addLeadingSlash,
   removeTrailingSlash,
```

Comment:
we probably want to move all those methods as well for consistency reasons it's weird if they all live in different packages